lmp/bb-config.sh

```diff
@@ -72,6 +72,7 @@ cat << EOFEOF >> conf/local.conf
 CONNECTIVITY_CHECK_URIS = "https://www.google.com/"
 
 ACCEPT_FSL_EULA = "1"
+LICENSE_FLAGS_ACCEPTED:append:rpi = " synaptics-killswitch"
```

Comment:
On other thing I just remembered, this change will be used on every factory, and some might still not be using the new syntax, so we need to do something like done for other variables, like:

```
if [ "$CONF_VERSION" == "2" ]; then
        cat << EOFEOF >> conf/local.conf
LICENSE_FLAGS_ACCEPTED:append:rpi = " synaptics-killswitch"
EOFEOF
fi
```

Or just reuse one of the if/else blocks that are already there.

Reply:
And it is fine to only cover the new syntax as we know this is not an issue on older meta-rpi layers.

Reply:
> On other thing I just remembered, this change will be used on every factory, and some might still not be using the new syntax, so we need to do something like done for other variables, like:
> 
> ```
> if [ "$CONF_VERSION" == "2" ]; then
>         cat << EOFEOF >> conf/local.conf
> LICENSE_FLAGS_ACCEPTED:append:rpi = " synaptics-killswitch"
> EOFEOF
> fi
> ```
> 
> Or just reuse one of the if/else blocks that are already there.

I had already thought about suggesting deleting the old syntax but now I understand why it still exists.